app/mailers/portal_mailer.rb

```diff
@@ -0,0 +1,13 @@
+class PortalMailer < ApplicationMailer
+
+  # @param hideRecipients [Boolean], recipients [Array<String>], reply_to [Array<String>], subject [String], content [String]
+  # @return [ActionMailer::MessageDelivery] a mail object with the given parameters.
+  def generic_email(hideRecipients, recipients, reply_to, subject, content)
```

Comment:
I know, we haven't any conventions about this, but afaik it is very common in ruby to naming boolean flags with a question mark suffix. 
`hideRecipients -> hide_recipients?`